src/controller/Controller.js

```diff
@@ -0,0 +1,54 @@
+import InputView from '../view/InputView.js';
+import Car from '../model/Car.js';
+import OutputView from '../view/OutputView.js';
+
+export default class Controller {
+  constructor() {
+    this.carList = [];
+    this.MAX_MOVE = 0;
+  }
+
+  /**
+   * 자동차 이름 목록을 통해 자동차 객체를 생성합니다.
+   * @param {string[]} [자동차 이름 목록]
+   */
+  createCar(carNameList) {
+    carNameList.forEach((carName) => {
+      this.carList.push(new Car(carName));
+    });
```

Comment:
```suggestion
    return carNameList.map((carName) => new Car(carName));
```
이렇게 바꾼다면, 간결해지고 기존 배열에 변화를 주는 일을 방지해서 더 코드가 명확해질 것 같습니다.
"불변성 유지" 키워드로 찾아보시면 도움이 될 것 같아요!

Reply:
와... 저도 이 부분에 대해서 어떻게 값을 반환해주지..? 라는 고민을 했었는데 map함수를 망각하고 있었네요 의견 감사합니다! 다음 미션 때 반영해봐야겠어요 :)